scripts/test_whisper_daemon.py

```
@@ -0,0 +1,251 @@
+        """Test that invalid SSL certificates are rejected"""
+        # This test should FAIL initially because SSL verification is disabled
+        
+        request = {
+            "action": "transcribe", 
+            "url": "https://self-signed.badssl.com/",  # Known bad certificate
+            "id": "test-invalid-cert"
+        }
+        
+        # Mock whisper model
+        with patch.object(self.daemon, 'model', None):
+            with patch('whisper.load_model') as mock_load_model:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)